Docs/source/running_cpp/parameters.rst

```diff
@@ -44,6 +44,17 @@ Overall simulation parameters
 * ``warpx.verbose`` (`0` or `1`)
     Controls how much information is printed to the terminal, when running WarpX.
 
+* ``warpx.random_seed`` (`string` or `int` >= 1) optional
```

Comment:
This is being picky, but I would say `int` > 0, since 1 would be a valid value. The same below.

Reply:
@dpgrote Sorry I didn't get what you mean, you mean change `int >= 1` to `int > 0`?